LZW/Tree.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace LZW;
+
+using System.Collections.Generic;
+
+/// <summary>
+/// This is a tree node.
+/// </summary>
+public class Node
+{
+    private char value;
+    private int isEndOfWord;
+    private List<Node> listOfTrees;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Node"/> class.
+    /// </summary>
+    /// <param name="value"> The value Of node. </param>
+    public Node(char value)
+    {
+        this.value = value;
+        this.listOfTrees = new ();
+    }
+
+    /// <summary>
+    /// Gets the value of the node.
+    /// </summary>
+    public char Value
+    {
+        get { return this.value; }
+    }
```

Comment:
```suggestion
    public char Value => this.value;
```
И тут Value могло бы быть автоматическим свойством.